src/torchcodec/decoders/_audio_decoder.py

```diff
@@ -25,10 +25,13 @@ def __init__(
         source: Union[str, Path, bytes, Tensor],
         *,
         stream_index: Optional[int] = None,
+        sample_rate: Optional[int] = None,
```

Comment:
We could also consider exposing this as `desired_sample_rate` parameter? I don't have a strong opinion, the docs would make it clear what this means in any case.